x/liquidity/types/querier.go

```diff
@@ -11,7 +11,7 @@ const (
 
 // QueryLiquidityPoolParams is the query parameters for 'custom/liquidity'
 type QueryLiquidityPoolParams struct {
-	PoolId uint64 `json:"pool_id" yaml:"pool_id"` // nolint:golint
+	PoolId uint64 `json:"pool_id" yaml:"pool_id"` // nolint:revive
```

Comment:
I think It should be `//nolint:revive`

Reply:
`Error: directive `// nolint:revive` should be written without leading space as `//nolint:revive` (nolintlint)`